generators/readme/src/index.ts

```diff
@@ -29,6 +29,9 @@ import type {
 import fs from 'fs';
 import path from 'path';
 import shelljs from 'shelljs';
+import Listr from 'listr';
+// @ts-ignore
```

Comment:
Maybe @ts-expect-error?

Reply:
It's just me being lazy and not providing the types. I think "ignore" is fine.